backend/src/main/java/com/huellapositiva/application/dto/ESALRequestDto.java

```diff
@@ -5,16 +5,81 @@
 import lombok.Builder;
 import lombok.Data;
 
+import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
 
 @Data
 @Builder
 @AllArgsConstructor
 public class ESALRequestDto {
 
     @Schema(
+            description = "Name of ESAL",
             example = "Canarias para todos"
     )
-    @NotEmpty
+    @NotBlank
+    @Pattern(regexp = "^[a-zA-Z ]+$")
     private final String name;
+
+    @Schema(
+            description = "ESAL's description",
+            example = "ESAL dedicada a ayudar en la recogida de alimentos."
+
+    )
+    @Size(max = 500)
+    private final String description;
+
+    @Schema(
+            description = "ESAL's website",
+            example = "https://www.foo-bar.com/"
+    )
+    @NotEmpty
+    @Size(max = 255)
+    private final String website;
+
+    @Schema(
+            description = "Whether the ESAL is registered as an Volunteering Entity by Gobierno de Canarias",
+            example = "true"
+    )
+    @NotBlank
+    private final boolean registeredEntity;
+
+    @Schema(
+            description = "Entity type of the ESAL",
+            example = "Asociacion"
+    )
+    @NotBlank
+    private final String entityType;
+
+    @Schema(
+            description = "Whether the ESAL has accepted the Privacy Policy",
+            example = "true"
+    )
+    @NotBlank
+    private final boolean privacyPolicy;
+
+    @Schema(
+            description = "Whether the ESAL has accepted the Data Protection Policy",
+            example = "true"
+    )
+    @NotBlank
```

Comment:
boolean cannot be blank, there are many occurrences of this

Reply:
Ok, you are right